cmd/dep/status.go

```diff
@@ -273,6 +274,36 @@ func (cmd *statusCommand) Run(ctx *dep.Ctx, args []string) error {
 	return nil
 }
 
+func (cmd *statusCommand) runOld(ctx *dep.Ctx, args []string, p *dep.Project, sm gps.SourceManager, params gps.SolveParameters) error {
+	solver, err := gps.Prepare(params, sm)
+	if err != nil {
+		return errors.Wrap(err, "fastpath solver prepare")
+	}
+
+	solution, err := solver.Solve(context.TODO())
+	if err != nil {
+		return errors.Wrap(err, "runOld")
+	}
+
+	var oldLockProjects []gps.LockedProject
+	lockProjects := p.Lock.Projects()
+	solutionProjects := solution.Projects()
+
+	for _, sp := range solutionProjects {
+		for _, lp := range lockProjects {
+			spr, _, _ := gps.VersionComponentStrings(sp.Version())
+			lpr, _, _ := gps.VersionComponentStrings(lp.Version())
+
+			if spr != lpr {
+				oldLockProjects = append(oldLockProjects, lp)
+			}
+		}
+	}
+
+	// TODO: Print output
```

Comment:
You can print to `ctx.Out` for now. Maybe we would like to use some [`outputter`](https://github.com/golang/dep/blob/master/cmd/dep/status.go?utf8=%E2%9C%93#L87) later.
```golang
ctx.Out.Println("print some text to stdout")
```